src/core/balance-sheet.ts

```diff
@@ -61,7 +61,10 @@ const updatePendingLightningPayments = async () => {
       )
       const user = await User.findOne({ _id: ledger.resolveAccountId(account) })
       const userWallet = await WalletFactory({ user, logger })
-      await userWallet.updatePendingPayments()
+      await Wallets.updatePendingPayments({
+        walletId: userWallet.user.id as WalletId,
```

Comment:
if you only need the wallet Id you dont need line 62, just use `ledger.resolveAccountId(account)` to get the walletId